src/engine/structures.rs

```diff
@@ -178,6 +188,14 @@ impl<T: HasName + HasUuid> Table<T> {
             (Some(name_item), Some(uuid_item)) => vec![name_item, uuid_item],
         }
     }
+
```

Comment:
The consuming version was basically missing. So, the first two implementations only work on a reference of a `Table`. But calling `into_iter` on a non-reference `Table` was impossible so far.

The consuming version then can _exploit_ the consuming version of `Vec`'s `into_iter`